internal/nodes/avi_model_nodes.go

```diff
@@ -619,47 +619,45 @@ func (v *AviVsNode) CalculateCheckSum() {
 		return portproto[i].Name < portproto[j].Name
 	})
 
-	var dsChecksum, httppolChecksum, sniChecksum, sslkeyChecksum, l4policyChecksum, passthroughChecksum, vsvipChecksum uint32
+	var checksumStringSlice []string
 
 	for _, ds := range v.HTTPDSrefs {
-		dsChecksum += ds.GetCheckSum()
+		checksumStringSlice = append(checksumStringSlice, "HTTPDS"+ds.Name)
 	}
 
 	for _, httppol := range v.HttpPolicyRefs {
-		httppolChecksum += httppol.GetCheckSum()
-	}
-
-	for _, sninode := range v.SniNodes {
-		sniChecksum += sninode.GetCheckSum()
+		checksumStringSlice = append(checksumStringSlice, "HttpPolicy"+httppol.Name)
```

Comment:
As we are considering the sniNodes in the parent VS checksum anymore, lets check if update in the SNI VS does impact the model checksum. I think it should.